src/isomorphism/vf2.rs

```diff
@@ -41,6 +38,48 @@ use crate::iterators::NodeMap;
 
 type StablePyGraph<Ty> = StableGraph<PyObject, PyObject, Ty>;
 
+/// Returns the adjacency_matrix of a graph with `(i, j)` entry equal
+/// to number of edges from node `i` to node `j`.
+fn adjacency_matrix<Ty: EdgeType>(graph: &StablePyGraph<Ty>) -> Vec<usize> {
+    let n = graph.node_bound();
+    let mut matrix = vec![0; n * n];
```

Comment:
Because of #367, I would say using a sparse matrix or hashmap would be more appropriate if we stick with the adjacency matrix approach